packages/flame/src/Core/index.tsx

```diff
@@ -109,14 +98,16 @@ const FlameTheme: React.FunctionComponent<FlameThemeProps> = ({
 // WARNING!
 // This component will be deprecated in v2. Instead, use the link tag directly:
```

Comment:
Oops, we lied here 😉

Reply:
its a typo, we actually meant v3 🤔

Reply:
I think we meant `v2` but that's fine. Let's move it in `v3`.